src/trackers/TVC.py

```
@@ -147,14 +157,48 @@ async def upload(self, meta, disctype):
 
         await common.edit_torrent(meta, self.tracker, self.source_flag)
         await self.get_tmdb_data(meta)
-        if meta['category'] == 'TV':
-            cat_id = await self.get_cat_id(meta['genres'])
-        else:
-            cat_id = 44
-        # type_id = await self.get_type_id(meta['type'])
+        # load MediaInfo and extract audio languages first
+        with open(f"{meta['base_dir']}/tmp/{meta['uuid']}/MediaInfo.json", 'r', encoding='utf-8') as f:
```

Ruff (0.14.3)

Probable insecure usage of temporary file or directory: "/tmp/"

(S108)